code/game/mecha/coral_gen.dm

```diff
@@ -0,0 +1,45 @@
+/obj/item/mecha_parts/mecha_equipment/coral_generator
```

Comment:
This file belongs in the equipment folder somewhere.

Reply:
will move it

Reply:
moved